src/components/atoms/SmallStoryCardNoImageNumber/smallStoryCardNoImageNumber.m.css

```diff
@@ -0,0 +1,62 @@
+.base {
+  border-bottom: 3px solid var(--accent-color);
+  @media (min-width: 768px) {
+    border-bottom: none;
+  }
+}
+.image-wrapper {
+  /* @include image-aspect-ratio(4, 3); */
+  padding-top: 75%;
+  align-self: flex-start;
+}
+.contributor {
+  display: inline-flex;
+  color: var(--meta-color);
+}
+
+.text-wrapper {
+  display: inline-flex;
+  padding: var(--space2) 0;
+  @media (min-width: 768px) {
+    display: inline-flex;
+    padding: var(--space4) 0 var(--space3);
+  }
+}
+.card-content {
+  line-height: 1.25;
+}
+.headline {
+  padding-top: var(--space-2);
+  /* @include font-styles(h5); */
+  font-size: var(--base-medium);
```

Comment:
let this not be here, the font size should ideally be in headline.m.css

Reply:
done